src/main/resources/messages.json

```diff
@@ -306,6 +306,60 @@
         "label": "More info"
       },
       "confirmButton": null
+    },
+    {
+      "id": "premature",
+      "title": "An announcement before its time.",
+      "titleShort": "Not yet gone live.",
+      "description": "This announcement is not live. Too soon..",
+      "descriptionShort": "Premature.",
+      "messageType": "announcement",
+      "goLiveDate": "2999-01-01",
```

Comment:
This will fail with the new refactored filter structure. Let's merge #49 first, and then fix this with the new structure.